pom.xml

```diff
@@ -4,13 +4,13 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <artifactId>presto-root</artifactId>
-        <groupId>io.prestosql</groupId>
-        <version>340</version>
+        <artifactId>trino-root</artifactId>
+        <groupId>io.trino</groupId>
+        <version>356</version>
```

Comment:
The latest version is 357 and works fine after some testing.

Reply:
Changed both trino-root and trino-testing to 357, confirmed this works too.